src/wazuh_db/schema_upgrade_v11.sql

```diff
@@ -0,0 +1,33 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Febrary 1, 2023.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS sys_hwinfo_tmp (
```

Comment:
I'd use `_sys_hwinfo` as temporary name to be aligned with the previos upgrades.

Reply:
ok, I'll do it